lib/host/android_notifications.dart

```diff
@@ -60,3 +60,22 @@ abstract class PendingIntentFlag {
   /// Corresponds to `FLAG_ALLOW_UNSAFE_IMPLICIT_INTENT`.
   static const allowUnsafeImplicitIntent = 1 << 24;
 }
+
+/// For use in [AndroidIntent.action].
+///
+/// See: https://developer.android.com/reference/android/content/Intent#constants_1
+abstract class IntentAction {
+  /// Corresponds to `ACTION_VIEW`.
+  static const view = 'android.intent.action.VIEW';
+}
+
+/// For use in [AndroidIntent.flags].
+///
+/// See: https://developer.android.com/reference/android/content/Intent#constants_1
+abstract class IntentFlag {
```

Comment:
nit:
```suggestion
/// See: https://developer.android.com/reference/android/content/Intent#FLAG_ACTIVITY_BROUGHT_TO_FRONT
abstract class IntentFlag {
```

There's a ton of constants on that page, so going straight to one of the flags (the one that's currently first in the list) saves some scrolling.